Providers/Scripts/nxDSCLog.py

```diff
@@ -94,7 +94,7 @@ def __init__(self, path=LogFile):
     def write(self, message):
         self.console.write(message)
         try:
-            file_handle = codecs.open(self.logpath, 'a', 'utf8')
+            file_handle = codecs.open(self.logpath, 'a+', 'utf8')
```

Comment:
we are only opening the file for writing. (Mode 'a' should do it). a + is for read write access

Reply:
Ok. Will remove the +